src/style.css

```diff
@@ -1,3 +1,113 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  padding-top: 1px;
+
+  --header-height: 60px;
+  --font-size: 12px;
+  --link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  border: 0;
+
+  /* max-width: 1200px;
+  min-width: 1024px; */
+
+  width: 100%;
+  box-sizing: border-box;
+  align-items: center;
+  outline: none;
```

Comment:
Don't fix container size (if there is no such a requirement). Here, the width is set to 100% which might be unnecessary if the 'header' is already a block-level element, which by default takes up 100% width.